### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by alexiscyber14**
Requirements:
Create a branch(i created the feature branch)
On the feature branch, create an HTML heading element.
On the feature branch create a CSS file and give the heading's inner text a color.
On the feature branch create a test.md file and tell git to ignore it.

UPDATES.
I updated the repository and I was able to remove the test.md file with git ignore

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# This file shoud be ignored by git
```
Comment (leonardodiasb) [inline test.md:1]: - [ ] As you mentioned in this file, it should be ignored by the `.gitignore` file. Please remove this file from this repository.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @alexiscyber14, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Linters set up correctly. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] Your Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail. Kindly edit those to make them look more professional so the code reviewer can understand better what to review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (alexiscyber14) [reply test.md]: Thank you!
I have resolved that issue

Comment (alexiscyber14) [reply test.md]: I have updated it. Thank you

File: README.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
```
Comment (levy002) [inline README.md:63]: - [OPTIONAL] Kindly make sure to add the project name in order to make your README professional.

**Review by levy002 — APPROVED** [review]
## Hello @alexiscyber14 
### I'm Levy, your PR reviewer

 Excellent, you nailed it :sunglasses: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 



## Highlights
- Good job for using correct git-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**